app/commands/exercise/representation/find_example_submissions.rb

```diff
@@ -10,10 +10,11 @@ def call
 
     loop do
       submissions = representation.
-        submission_representation_submissions.
+        submission_representations.
+        includes(:submission).
         order(id: :desc).
         page(page).
-        per(NUM_EXAMPLES * 2)
+        per(NUM_EXAMPLES * 2).map(&:submission)
```

Comment:
This is 2ms, not 20s, so a good 4 orders of magnitude faster. Got to love SQL! 🙂